src/urllib3/response.py

```diff
@@ -148,6 +153,22 @@ def flush(self) -> bytes:
             return b""
 
 
+if zstd is not None:
+
+    class ZstdDecoder(ContentDecoder):
+        def __init__(self) -> None:
+            self._obj = zstd.ZstdDecompressor().decompressobj()
+
+        def decompress(self, data: bytes) -> bytes:
+            if data == b"":
+                return b""
+            else:
+                return self._obj.decompress(data)  # type: ignore[no-any-return]
```

Comment:
For consistency with other decoders, can we please use an early return and check for all falsy values?

```suggestion
            if not data:
                return data

            return self._obj.decompress(data)  # type: ignore[no-any-return]
```